packages/api/src/hooks/useRemoteConfig.ts

```diff
@@ -15,7 +15,7 @@ const remoteConfigQuery = async function () {
     return response.json();
 };
 
-function useRemoteConfig(enabled = false) {
+function useRemoteConfig(enabled = true) {
```

Comment:
isn't it better to pass the true to this instead making the default value true?
Asking this because we are not sure about the sideeffect of this change on other places that we are using it.

Reply:
In `deriv/api-v2` package we even don't have `enabled` parameter and it's always `true`, so I tried to make the similar approach, but ok, I set `false` by default and call all hooks with `true` for now